packages/pg/lib/crypto/utils.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const nodeCrypto = require('crypto')
+
+module.exports = {
+  postgresMd5PasswordHash,
+  randomBytes,
+  deriveKey,
+  sha256,
+  hmacSha256,
+  md5,
+}
+
+/**
+ * The Web Crypto API - grabbed from the Node.js library or the global
+ * @type Crypto
+ */
+const webCrypto = nodeCrypto.webcrypto || globalThis.crypto
```

Comment:
This is basically the decision point between using internal Node.js APIs and built-in serverless (i.e. Cloudflare) ones.